git-hub

```diff
@@ -87,15 +103,292 @@ def push():
+            else:               # checks for all other parameters
+                temp_user = sub_dictionary['user']
+                temp_branch = sub_dictionary["branch"]
+                temp_comment = sub_dictionary["comment"]
+                temp_label = sub_dictionary["labels"]
+                if label.upper() in temp_label.upper() and comment.upper() in temp_comment.upper() and user.upper() in temp_user.upper() and branch.upper() in temp_branch.upper():
+                    appeared_before = True
+                    if(open_or_closed):
+                        q.put((sub_dictionary['modified'], '{0} \x1b[0;32;40m {1} \x1b[0m {2}/{3} {4}'.format(sub_keys, "O", sub_dictionary['user'], sub_dictionary['branch'], sub_dictionary['comment'])))
+                    else:
+                        q.put((sub_dictionary['modified'], '{0} \x1b[0;31;40m {1} \x1b[0m {2}/{3} {4}'.format(sub_keys, "C", sub_dictionary['user'], sub_dictionary['branch'], sub_dictionary['comment'])))
+    return appeared_before, q
+
+
+def search(keyword, user, comment, number, branch, opened, label, sort):
+    """Searches open and closed pull request comments for specified keyword.
+
+    Opens pull-requests.toml file in .git folder to fetch pull requests.
+
+    Parameters
+    ----------
+    keyword : string
+        Searches if any part of user, branch, comment, or number match
+    user : string
+        Search by PR username
+    comment : string
+        Search by PR comment
+    number : string
+        Search by PR number
+    branch : string
+        Search by PR branch
+    """
+    try:
+        open_prs = False
+        closed_prs = False
+        path_git = find_path_to_git()
+        path_github = pjoin(path_git, 'git-hub')
+        path_prs = pjoin(path_github, 'pull-requests.toml')
+        f = open(path_prs, "r")
+
+        pr_dict = toml.load(f)      # fetches toml file and creates a dictionary
+        open_dict = pr_dict['open pull requests']
+        closed_dict = pr_dict['closed pull requests']
+
+        if(not opened or opened.lower() == 'open'):
+            open_prs, q = findMatch(True, open_dict, keyword, user, comment, number, branch, label, False, False)
+            print_in_order(sort, q)
+        if(not opened or opened.lower() == 'closed'):
+            closed_prs, q = findMatch(False, closed_dict, keyword, user, comment, number, branch, label, False, open_prs)
+            print_in_order(sort, q)
+        if not open_prs and not closed_prs:     # outputs if keyword was not contained in pull requets comments
+            click.echo("Could not find in pull requests. Update your pull requests with 'git hub sync' and try again.")
+    except (OSError, IOError) as e:
+        # if pull-requests.toml hasnt been created yet calls sync and then reties to fetch
+        sync()
+        search(keyword, user, comment, number, branch, status)
+
+
+def parse_to_toml(username, repo, pr, token, open_or_closed):
```

Comment:
Can we come up with a better naming here?  What is being parsed into toml, and why?